oci/table_oci_core_local_peering_gateway.go

```diff
@@ -0,0 +1,249 @@
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "lifecycle_state",
+				Description: "The LPG's current lifecycle state.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "The date and time the LPG was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+
+			// other columns
+			{
+				Name:        "is_cross_tenancy_peering",
+				Description: "Whether the VCN at the other end of the peering is in a different tenancy.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "peer_advertised_cidr",
+				Description: "The smallest aggregate CIDR that contains all the CIDR routes advertised by the VCN at the other end of the peering from this LPG.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "peering_status_details",
+				Description: "Additional information regarding the peering status.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "route_table_id",
+				Description: "The OCID of the route table the LPG is using.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+
+			// json fields
+			{
+				Name:        "peering_status",
+				Description: "Whether the LPG is peered with another LPG.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "peer_advertised_cidr_details",
+				Description: "The specific ranges of IP addresses available on or via the VCN at the other end of the peering from this LPG.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// tags
+			{
+				Name:        "defined_tags",
+				Description: ColumnDescriptionDefinedTags,
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "freeform_tags",
+				Description: ColumnDescriptionFreefromTags,
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Standard Steampipe columns
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(gatewayTags),
+			},
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+
+			// Standard OCI columns
+			{
+				Name:        "compartment_id",
+				Description: ColumnDescriptionCompartment,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CompartmentId"),
+			},
+			{
+				Name:        "tenant_id",
+				Description: ColumnDescriptionTenant,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getTenantId,
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listPeeringGateway(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func listCoreLocalPeeringGateways(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```
Change wherever required.